composer.json

```diff
@@ -25,11 +25,9 @@
     ],
     "require": {
         "php": "^7.3|^8.0",
-        "aic/data-hub-foundation": "dev-laravel-8-support",
+        "aic/data-hub-foundation": "1.0",
```

Comment:
Should this be "^1.0" so it will pull minor updates? Or should we be explicit about which version we're using?

Reply:
Good point, we don't want to have to update every package with every minor release!